packages/manager/modules/web-paas/src/index.less

```diff
@@ -0,0 +1,66 @@
+@import 'flag-icon-css/less/flag-icon.less';
+@import (less) 'flag-icon-css/css/flag-icon.min.css';
+
+/* ==================================================
+  =            Add some flags definitions            =
+  ================================================== */
+.flag-icon {
+  border-radius: 3px;
```

Comment:
Use flag icons provided by ui-kit. remove flag-icon dependency.

Reply:
I dont see `oui-flag` being used in any modules and even if we have to use it, we have to replace `flag-icon` dependency everywhere. Also `regionService` must be exported outside cloud universe components so that it can be used universally.Might be better if we take this as a refactoring ticket ?